Riot/Modules/Application/AppCoordinator.swift

```diff
@@ -131,7 +131,7 @@ final class AppCoordinator: NSObject, AppCoordinatorType {
     
     private func addSideMenu() {
         let appInfo = AppInfo.current
-        let coordinatorParameters = SideMenuCoordinatorParameters(appNavigator: self.appNavigator, userSessionsService: self.userSessionsService, appInfo: appInfo)
+        let coordinatorParameters = SideMenuCoordinatorParameters(appNavigator: self.appNavigator, appCoordinator: self, userSessionsService: self.userSessionsService, appInfo: appInfo)
```

Comment:
The `AppCoordinator` should not be injected, other coordinators should not know is existence. It's adding an unnecessary object relation and avoid reusability.